RadixWallet/Features/ClaimWallet/ClaimWallet+View.swift

```diff
@@ -38,6 +38,12 @@ extension ClaimWallet {
 					Spacer()
 
 					VStack {
+						if viewStore.failedToReclaim {
+							Text("Failed to transfer ownership")
+								.foregroundColor(.app.error)
+								.textStyle(.secondaryHeader)
+						}
```

Comment:
We have no design for this, but on the other hand it is probably impossible to get into the state where we show it - if ProfileStore fails to claim the profile.